sdk/search/search-documents/samples-dev/bufferedSenderAutoFlushSize.ts

```diff
@@ -5,22 +5,26 @@ import {
   GeographyPoint,
   SearchIndexClient
 } from "@azure/search-documents";
-import { createIndex, documentKeyRetriever, WAIT_TIME } from "../../utils/setup";
-import { Hotel } from "../../utils/interfaces";
+import { createIndex, documentKeyRetriever, WAIT_TIME } from "./setup";
+import { Hotel } from "./interfaces";
 import { delay } from "@azure/core-http";
 import * as dotenv from "dotenv";
 dotenv.config();
 
+/**
```

Comment:
Such a small thing, but I think these are nicer if they are at/near the top of the file.

Reply:
Done